README.md

```diff
@@ -21,7 +21,7 @@ Before you can run the frontend app against the backend you will need to configu
 
 ### Set the Environment variables
 
-Create a copy of the .env.sample file and rename it .env and fill in below values:
+Create a file and name it .env and fill in below values:
```

Comment:
I've just removed this commit now, I think we wanted to make the process easier because at the moment every time I have to copy these values, then manually compare what's in the .env.sample file to check which to keep. But I don't have the brain space at the moment to work out the better way, so have just removed this commit